versioning/BRM/extentmap.cpp

```diff
@@ -1060,160 +1062,127 @@ void ExtentMap::mergeExtentsMaxMin(CPMaxMinMergeMap_t& cpMap, bool useLock)
         grabEMIndex(WRITE);
     }
 
-    int entries = fEMShminfo->allocdSize / sizeof(struct EMEntry);
-
-    for (int i = 0; i < entries; i++)  			// loop through all extents
+    for (auto emIt = fExtentMapRBTree->begin(), end = fExtentMapRBTree->end(); emIt != end; ++emIt)
```

Comment:
This mergeExtentsMaxMin() method' loop should iterate over cpMap and use findByLBID() to reduce computation complexity.